prometheus/testutil/promlint/promlint.go

```diff
@@ -0,0 +1,367 @@
+// Copyright 2017 The Prometheus Authors
```

Comment:
It doesn't matter a lot, but we use to fill in the current year if we create a new file.